src/framework/openapi.spec.loader.ts

```diff
@@ -106,7 +106,7 @@ export class OpenApiSpecLoader {
     // instead create our own syntax that is compatible with express' pathToRegex
     // /{path}* => /:path*)
     // /{path}(*) => /:path*)
-    const pass1 = part.replace(/\/{([^\*]+)}\({0,1}(\*)\){0,1}/g, '/:$1$2');
+    const pass1 = part.replace(/\/{([^{}]+)}\({0,1}(\*)\){0,1}/g, '/:$1$2');
```

Comment:
This seems to work, thanks for looking into it! 

One small suggestion so the regex is more in line with 
https://github.com/cdimascio/express-openapi-validator/blob/631fb7b548f8c86f491c79dbf20d447b5305d9fc/src/framework/openapi.spec.loader.ts#L112
```suggestion
    const pass1 = part.replace(/\/{([^}]+)}\({0,1}(\*)\){0,1}/g, '/:$1$2');
```

Reply:
Yes, your solution looks better 👍